frontend/src/store/FamLoginUserState.ts

```diff
@@ -150,6 +162,14 @@ const cacheUserAccess = async () => {
     }
 };
 
+const delegatedCachedData = (application_id: number): FamRoleDto[] => {
```

Comment:
let's rename this to `getCachedAppRoles`

Reply:
Thanks. Changed as suggested.